strongbox-resources/strongbox-common-resources/src/main/resources/logback-spring.xml

```diff
@@ -25,7 +25,7 @@
             <filter class="org.carlspring.strongbox.log.JaninoClassLoaderLogDeclineFilter"/>
     
             <encoder>
-                <pattern>%d{HH:mm:ss.SSS dd-MM-yyyy} | %-5.5p | %-20.20t | %-50.50logger{50} | %m%n</pattern>
+                <pattern>%d{HH:mm:ss.SSS} | %.-1level | %-10.10t | %-37.37logger{37} | %m%n</pattern>
```

Comment:
Why have you dropped the `dd-MM-yyyy` part from the logging pattern? Is this for all loggers?

Reply:
the thing is that I found the date redundant in log files, because we have `RollingFileAppender` based on `SizeAndTimeBasedRollingPolicy`, so the date could be found in file name. The advantage of this is to have less line length in logs, so it fits the width of the screen.

Reply:
Is the default name of the (current) log file also timestamped?

Reply:
current log file is not timestamped.

Reply:
Added #1976 .